anime_downloader/commands/ezdl.py

```diff
@@ -145,17 +151,18 @@ def command(ctx, anime_url, episode_range, player,
                             fallback_qualities=fallback_qualities)
             # I have yet to investigate all errors this can output
             # No sources found gives error which exits the script
-            except:
+            except BaseException:
                 continue
 
             logger.debug('Found anime: {}'.format(anime.title))
 
             try:
                 animes = util.parse_ep_str(anime, str(episode_range))
             except RuntimeError:
-                logger.error('No episode found with index {}'.format(episode_range))
+                logger.error(
+                    'No episode found with index {}'.format(episode_range))
                 continue
-            except:
+            except BaseException:
```

Comment:
This is pointless, did you aggressively autopep8?

Reply:
yes